website/config/routes.js

```diff
@@ -275,6 +275,15 @@ module.exports.routes = {
   },
 
 
+  'GET /demo/okta-webflow': {
+    action: 'demo/view-okta-webflow',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+      optimizeForAppleWebview: true,
```

Comment:
Makes sense.  We'll end up with more peronsalization in this view action, but since this view local controls something in the layout, I think it makes sense for it to live in config/routes.js.